src/content/docs/tutorial-app-performance/reduce-javascript-errors.mdx

```diff
@@ -13,56 +13,53 @@ import browserJsErrorCharts from 'images/browser_screenshot-crop_error-overview.
 
 import browserSingleErrorOverview from 'images/browser_screenshot-crop_error-eventlog-stacktrace-overview.webp'
 
-JavaScript errors can have significant negative impacts on how customers use your site and perceive your brand. From causing user frustration to damaging brand perception, you need to find and fix JavaScript errors as soon as they occur to help maintain site performance and keep your customers happy. New Relic makes pin-pointing these errors straightforward no matter the number or the size, helping you get your site up and running faster and more efficiently.
+JavaScript errors affect how customers use your site and their perception of your brand. You should prioritize fixing JavaScript errors as soon as they occur to help maintain site performance and keep your customers happy. 
 
-## Finding JavaScript errors [#find-errors] 
+## Objectives [#objectives]
+
+This tutorial walks you through finding and resolve JavaScript errors. By the end of the tutorial, you will have these tasks completed:
+
+* Finding JavaScript errors information in New Relic
+* Applying our recommendations when prioritizing errors
+* Using event logs to resolve JavaScript errors
+
+## Find JavaScript errors [#find-errors] 
+
+From the browser monitoring **Summary** page, two charts can indicate you have JavaScript errors: **Error Rate** and **Top 10 error messages**. Selecting either chart takes you to the **JS errors** page, which gives you a more detailed view of your site'sJavaScript errors. 
+
+* If a customer or internal user reports something that could be a JavaScript error, we recommend starting your search at the **Summary** or **JS errors** pages. 
+* If you want to proactively monitor your site for errors, checking the **Summary** page is adequate most of the time. 
+
+## Determine what errors matter [#determine-errors]
+
+Once you've discovered some JavaScript errors, you need to prioritize your fixes. Prioritizing errors is complex, and can be based on frequency of the error, severity, regions affected, and other factors. Let's use an example to walk you through how to prioritize JavaScript errors. 
```

Comment:
Maybe we can cut out "is complex" here? Sounds a little discouraging potentially.